spec/acceptance/suites/default/05_generate_types_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper_acceptance'
+
+describe 'incron driven puppet generate types'  do
+  hosts_with_role(hosts, 'master').each do |host|
+    context "on #{host}" do
+      let(:environment_path) { host.puppet[:environmentpath] }
+      let(:incron_cache) {
+        "#{environment_path}/production/.resource_types"
```

Comment:
Where does this file get generated? Can we name it to something that implies it's part of an incron table?

Reply:
This is what `puppet generate types` generates. It cannot be changed.

Reply:
:+1: